tests/test_pytest/test_output_checkers.py

```diff
@@ -0,0 +1,53 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test all Output Checker plugins."""
+import pytest
+from PIL import Image
+import numpy as np
+
+from geoips.interfaces import output_checkers
+from geoips.commandline import log_setup
+
+
+log_setup.setup_logging()
+image = output_checkers.get_plugin("image")
+savedir = str(__file__).replace(
+    "geoips/tests/test_pytest/test_output_checkers.py", "test_data/test_images/pytest/")
+thresholds = ["lenient", "medium", "strict"]
```

Comment:
Didn't we change this to floats?  Just pass in the "threshold" argument value to pixelmatch?

Reply:
This has yet to be merged in.

Reply:
This was merged an I've since updated this.